tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
Are we sure about this name? Do you think we should make it be cockpit-packagekit ... to follow along with cockpit-ostree ... and with the assumption that other ways of updating a system will show up?